mitodl/ocw-hugo-themes
# feat: compact design

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-hugo-themes/issues/765

#### What's this PR do?
- Creates a new theme `course-v2` and implements new designs as provided in the issue ticket.

#### How should this be manually tested?
- Checkout this branch
- In your local ocw course config (having env: `COURSE_HUGO_CONFIG_PATH`), add/rename `course` to `course-v2` to use this new theme instead of the previous one.
- Build any course locally.
- Verify that the UI is according to the provided designs for the course home page and course interior pages.
- Repeat this for:
    - as many courses as you can :) I'm also currently testing.
    - multiple browsers (chrome, firefox, safari etc)
    - multiple screen sizes (desktop, laptop, tablet, mobile etc)
- **Important:** Verify that the previous/old design of `course` and `www` still works as expected and there are no side-effects/impacts. 

#### Screenshots
- Please see this [deployed course](https://ocw-hugo-themes-course-pr-784--ocw-next.netlify.app/).
